pkg/cmd/server/api/types.go

```diff
@@ -144,6 +144,24 @@ type ProjectConfig struct {
 	// It is in the format namespace/template and it is optional.
 	// If it is not specified, a default template is used.
 	ProjectRequestTemplate string
+
+	// SecurityAllocator controls the automatic allocation of UIDs and MCS labels to a project. If nil, allocation is disabled.
```

Comment:
Not now.  If you have an SCC policy that says "only run pods with UIDS in this range" and you turn the allocator off, you won't run until an admin sets a range.  Which is reasonable for some of the scenarios we've discussed.